services/web/project/service/github_pull_request_extrator_service.py

```diff
@@ -3,20 +3,28 @@
 from ..model import Github_pull_request
 from ..repository import github_user_repository as user_repository
 from ..repository import github_repository_repository as repository_repository
-from . import github_extractor_service as user_service
+from . import github_user_service as user_service
 from . import github_repository_extrator_service as repository_service
 
 
-def save_pulls_by_repo(user, repo_name):
-    g = instance_service.get_available_instance()
-    repo_path = user + '/' +repo_name
-    repo = g.get_repo(repo_path , lazy=True).parent
+def save_pulls_by_repo_name(user, repo_name):
+    repo = get_repo(user, repo_name)
+    save_pulls_by_repo(repo)
 
+def save_pulls_by_repo(repo):
     pulls = repo.get_pulls(state='all')
     for pygithub_pull in pulls:
         github_pull_request = from_pygithub_object(pygithub_pull, repo)
         github_pull_request_repository.save(github_pull_request)
 
+def save_from_issue(issue):
+    pull = issue. as_pull_request()
+    repo= issue.repository
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'repo' is assigned to but never used (F841)](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5469380)